liblangutil/ErrorReporter.h

```diff
@@ -118,6 +118,9 @@ class ErrorReporter
 		return m_errorCount > 0;
 	}
 
+	// @returns true if error shouldn't be stored
+	bool checkForExcessiveErrors(Error::Type _type);
```

Comment:
Did you take into account that this increments the counter? It would probably better to add another function that just returns whether the next error would hit the limit or something like that.

Reply:
> Did you take into account that this increments the counter? 

Completely missed this - thanks. Will address in next iteration.